src/test-apps/ToolCommonOptions.h

```diff
@@ -25,6 +25,8 @@
 #ifndef TOOLCOMMONOPTIONS_H_
 #define TOOLCOMMONOPTIONS_H_
 
+#include <vector>
```

Comment:
Is this code ever included in RTOS-based, on-device tests?

Reply:
It is not.